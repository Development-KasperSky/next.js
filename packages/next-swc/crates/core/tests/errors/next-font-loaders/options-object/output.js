import a from "@next/font/google?ABeeZee;{}";
import a from "@next/font/google?ABeeZee;{}";
import a from "@next/font/google?ABeeZee;{}";
import a from "@next/font/google?ABeeZee;{}";
const a = fn({
    10: 'hello'
});
const a = fn({
    variant: [
        i1
    ]
});
const a = fn({
    variant: ()=>{}
});
const a = fn({
    ...{}
});
