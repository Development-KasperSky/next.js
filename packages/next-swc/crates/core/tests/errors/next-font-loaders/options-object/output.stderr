
  x Unexpected object key type
   ,-[input.js:4:1]
 4 | const a = ABeeZee({ 10: 'hello' })
   :                     ^^
   `----

  x Unexpected value
   ,-[input.js:7:1]
 7 | const a = ABeeZee({ variant: [i1] })
   :                               ^^
   `----

  x Unexpected value
    ,-[input.js:10:1]
 10 | const a = ABeeZee({ variant: () => {} })
    :                              ^^^^^^^^
    `----

  x Unexpected spread
    ,-[input.js:13:1]
 13 | const a = ABeeZee({ ...{} })
    :                     ^^^
    `----
